spinalcordtoolbox/scripts/sct_process_segmentation.py

```diff
@@ -406,6 +433,23 @@ def main(argv=None):
                                                             perlevel=perlevel, vert_level=fname_vert_levels,
                                                             group_funcs=group_funcs)
     metrics_agg_merged = merge_dict(metrics_agg)
+    # Normalize CSA values (MEAN(area))
+    if arguments.normalize is not None:
+        path_model = os.path.join(__sct_dir__, 'spinalcordtoolbox', 'data', 'csa_normalization_models',
+                                  '_'.join(sorted(arguments.normalize.columns)) + '.csv')
+        if not os.path.isfile(path_model):
+            raise parser.error('Invalid choice of predictors in -normalize. Please specify sex and brain-volume or sex, brain-volume and thalamus-volume.')
+        # Get normalization model
+        # Models are generated with https://github.com/sct-pipeline/ukbiobank-spinalcord-csa/blob/master/pipeline_ukbiobank/cli/compute_stats.py
+        # TODO update link with release tag.
+        data_predictors = pd.read_csv(path_model, index_col=0)
+        # Add interaction term
+        arguments.normalize['inter-BV_sex'] = arguments.normalize['brain-volume']*arguments.normalize['sex']
+
+        for line in metrics_agg_merged:
+            # Normalize CSA value and replace in metrics_agg_merged
+            metrics_agg_merged[line]['MEAN(area)'] = normalize_csa(metrics_agg_merged[line]['MEAN(area)'], data_predictors, arguments.normalize)
```

Comment:
Nitpick: I think this is a shorter way of writing the same thing:

```suggestion
        for line in metrics_agg_merged.values():
            line['MEAN(area)'] = normalize_csa(line['MEAN(area)'], data_predictors, arguments.normalize)
```

(I also removed the comment since it might also be a bit self-explanatory.)